package.json

```diff
@@ -67,7 +67,7 @@
   },
   "homepage": "https://github.com/DataDog/dd-trace-js#readme",
   "engines": {
-    "node": ">=18"
+    "node": ">=14"
```

Comment:
This check and checks on the dd-trace version are also in a few places in code which might need to be updated as well which I don't see in this PR.

Reply:
The only place `engines` appears to be used in code is `scripts/preinstall.js`, which doesn't do anything based on particular version, just check compat, so it should be fine.

The only places the Node.js version is inspected appear to be legitimate checks on the version, not taking into account the dd-trace version at all.

The only places the tracer version (e.g. also DD_MAJOR and friends) is used seem to be places where we've done in-code breaking-change switches, which shouldn't be changed by this PR.

Reply:
Perfect then, just wanted to make sure nothing was missed 👍